vocabularies/georesources-report-types.ttl

```diff
@@ -31,6 +34,34 @@ grt:pa-22A a skos:Concept ;
     skos:prefLabel "Notice of Completion of seismic survey or scientific or technical survey"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
 
+grt:pggd-01 a skos:Concept ;
+    skos:definition "A notice to inform the Department that a petroleum well or bore will be drilled."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/georesource-report> ;
+    skos:prefLabel "Notice of intention to drill a petroleum well or bore"@en ;

```

Comment:
Do we want to keep "well or bore" or introduce "borehole" here.

Reply:
_We_ probably want to introduce Borehole. But section 29 of the Regulation refers to it specifically as a "petroleum well or bore"